### ISSUE TITLE
Get geometry of the Swiss Camp 10 m station

### ISSUE THREAD

**Issue by BaptisteVandecrux** [GEUS-Glaciology-and-Climate/GC-Net-level-1-data-processing#48]
What is the distance between the sonic ranger and the top of the mast?
Which level is the top one?

**Comment by houtzd** [comment]
I don't think this was ever measured at least in the last 10 years. Unfortunately the 10m tower fell down last year. We never had a chance to re-drill it in 2020 to lower it.

**Comment by BaptisteVandecrux** [comment]
`Unfortunately the 10m tower fell down last year. `
On which date was that seen?
Was the logger files collected back then? The last logger files for SWC are from 2019 (see #76 )
